ui/components/app/network-display/index.scss

```diff
@@ -35,6 +35,11 @@
     background-color: lighten($dodger-blue, 35%);
   }
 
+  &.chip {
```

Comment:
Ideally it shouldn't be necessary to qualify these styles with `.chip`. As I understand it, Chip is a more abstract component than NetworkDisplay. But `ui` styles get imported after `app` styles, so because Chip defines a `margin` property, that will override anything we put here. We need `.chip` so that these styles will win.

Reply:
Yeah, I opted for alphabetical ordering of imports before fully understanding the impact it would have. I do prefer us to have *some* scheme for ordering imports, though. Increasing specificity is fine.